docs/getting-started/prefect-explained.md

```diff
@@ -0,0 +1,122 @@
+
+![Cloud Dashboard](/img/ui/cloud-dashboard.png)
+
+Here are the three major concepts you need to know to build a production workflow with Prefect:
+
+- Flows
+- Deployments
+- Work pools
+
+## Flows
+
+A Prefect [flow](/concepts/flows/) is a Python function that defines a workflow and is decorated by `@flow`.
+When this function runs, Prefect can track the state of the workflow and visualize it in the UI.
+
+As shown below, when a flow is referenced as the entrypoint in a deployment it can be scheduled.
+
+![Dependency graph in the UI](/img/ui/dependency-graph.png)
+
+A flow is defined in Python code, stored in the location specified in a deployment, and represented on the server.
+
+## Work pools
+
+A [work pool](/concepts/work-pools/) provides presets for infrastructure.
+The work pool type could be a local subprocess, a Docker container, a Kubernetes cluster, or serverless cloud infrastructure such as AWS ECS, Google Cloud Run, VertexAI, or Azure Container Instances.
+Prefect Cloud provides two special work pool categories: Prefect managed work pools and push work pools on serverless cloud infrastructure (ECS, Google Cloud Run, or Azure Container Instances).
+
+Work pools are created in the UI or CLI and live on the server.
+
+## Deployments
+
+Creating a [deployment](/concepts/deployments/) makes your flow schedulable.
+The deployment contains metadata needed to run the flow in production.
+
+A [deployment specifies](/guides/prefect-deploy/) a single entrypoint flow function, a deployment name, a work pool, and the location of the flow code to be pulled at runtime.
+
+Flow code storage options include a Docker registry with the flow code baked into an image, git-based storage such as a GitHub repository, or cloud-provider storage such as AWS S3.
+
+Deployments are created in Python code with `deploy`, via the CLI interactively with `prefect deploy`, or applied from a `prefect.yaml` file. Deployments live on the server.
+
+To schedule a deployment run, you can create an ad-hoc run from the UI or CLI, add a programmatic schedule (such as cron), or create an [automation](#automations) so that the deployment runs in response to a trigger.
+
+## Other Prefect concepts
+
+The following Prefect-specific concepts are very commonly used, but are not essential to understand for all production workflows.
+
+- Tasks
+- Workers
+- Automations
+- Blocks
+
+### Tasks
+
+Python functions that are called within a flow can be decorated with `@task` to make them Prefect [tasks](/concepts/tasks/).
+Turning a function into a task unlocks a number of Prefect features, including:
+
+- logging
+- automatic retries
+- caching
+- result persistence
+- simple parallelization and concurrency
+
+Tasks are defined in Python code and represented on the server.
+Tasks currently must be called from a flow.
+
+### Workers
+
+Hybrid work pools require a worker to be running on your infrastructure in order to execute flows.
```

Comment:
This is the first appearance of the word `Hybrid`, I know it's in the UI but maybe a one-sentence breakdown on the work pool categories in the Work Pools section could work?